### DISCUSSION TITLE
Cannot add 2 facebook inboxes

### DISCUSSION THREAD

**Discussion by hongxy** [chatwoot/chatwoot#3276] [Self hosted] [answered]
I am running self-hosted chatwoot. I have 2 facebook pages, for different business, respectively. I would like to create 2 facebook inboxes (A/B) for each page.
I follow[ issue #2072](https://github.com/chatwoot/chatwoot/discussions/2072), and I can setup one facebook inbox, but I cannot add another facebook inbox.
I also follow https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/facebook-channel-setup and https://www.chatwoot.com/docs/product/channels/facebook, but again, I can only create 1 facebook inbox for 1 page (page A)

Situation:
- facebook page A and B are under different facebook user accounts
- in chatwoot .env file, there is only ONE place to put in app id/secret/token

Questions:
- what is the best way to have multiple facebook inboxes
- should I change facebook pages A and B under one facebook account?

**Comment by sojan-official** [comment, marked as answer]
@hongxy the .env file contains the app id, secret, token related to your Facebook app. You can configure any number of Facebook pages against your Facebook app

please refer : https://www.chatwoot.com/docs/product/channels/facebook
